File: app/src/main/java/network/devandroid/com/networklibrarycomparison/models/PhotosModel.java
package network.devandroid.com.networklibrarycomparison.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PhotosModel {

    @SerializedName("albumId")
    @Expose
    public int albumId;

    @SerializedName("id")
    @Expose
    public int id;

    @SerializedName("title")
    @Expose
    public String title;

    @SerializedName("url")
    @Expose
    public String url;

    @SerializedName("thumbnailUrl")
    @Expose
    public String thumbnailUrl;

}

File: app/src/main/java/network/devandroid/com/networklibrarycomparison/models/PhotosModel.kt
package network.devandroid.com.networklibrarycomparison.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PhotosModel {

    @SerializedName("albumId")
    @Expose
    var albumId: Int = 0

    @SerializedName("id")
    @Expose
    var id: Int = 0

    @SerializedName("title")
    @Expose
    var title: String? = null

    @SerializedName("url")
    @Expose
    var url: String? = null

    @SerializedName("thumbnailUrl")
    @Expose
    var thumbnailUrl: String? = null

}